temp.py

```diff
@@ -0,0 +1,2 @@
+import pdb
```

Comment:
**CODE_QUALITY**: Avoid using `pdb` in production code. If debugging is necessary, consider using logging or other non-intrusive methods.
```
import logging

logging.basicConfig(level=logging.INFO)
logging.info('hello')
```